CONTRIBUTING.md

```diff
@@ -2,13 +2,20 @@
 
 The balena CLI is an open source project and your contribution is welcome!
 
-After cloning this repository and running `npm install`, the CLI can be built with `npm run build`
-and executed with `./bin/balena`. In order to ease development:
+* Install the dependencies listed in the "NPM Installation" section of the
```

Comment:
You could make this a link direct to the "NPM Installation" section heading to make it even more convenient... `./INSTALL.md#npm-installation` (unless the general policy is not to link to section headings in case they get changed/edited)

Reply:
Done! 👍